backend/src/server.ts

```diff
@@ -33,11 +33,13 @@ export const getContext =
       database = serverDatabase,
       pubsub = serverPubsub,
       user = databaseUser,
+      config = CONFIG,
     }: {
       database?: ReturnType<typeof databaseContext>
       pubsub?: ReturnType<typeof pubsubContext>
       user?: (any) => Promise<any>
-    } = { database: serverDatabase, pubsub: serverPubsub, user: databaseUser },
+      config: typeof CONFIG
```

Comment:
i prefer this to be an optional parameter.

As I see now there is a default value - maybe sync how this behaves

Reply:
I suggest
```suggestion
      config?: () => typeof CONFIG
```

Reply:
I wouldn't do that as it changes the type of `Context` which we reference in our resolvers. We would have to check for nullish all the time.

Reply:
I  agree on the `?` part